File: Python/Human.py
# Mmebuat kelas HUman
class Human:
    # Membuat beberapa atirbut private
    # Nama Human
    __nama = ""
    # nik Human
    __nik = ""
    # jenis kelamin Human
    __gender = ""

    # Constructor with default initialization
    def __init__(self, nama="", nik="", gender=""):
        self.__nama = nama
        self.__nik = nik
        self.__gender = gender

    # Setter and Getter methods for private attributes
    # set Nama
    def setNama(self, nama):
        self.__nama = nama

    # Get Nama
    def getNama(self):
        return self.__nama

    # Set nik
    def setNIK(self, nik):
        self.__nik = nik

    # Get nik
    def getNik(self):
        return self.__nik

    # Set gender
    def setGender(self, gender):
        self.__gender = gender

    # Get gender
    def getGender(self):
        return self.__gender

File: C++/Human.cpp
#include "header.hh"

// Constructor
Human::Human()
{
    nama = "";
    nik = "";
    gender = "";
}

// Constructor with all atributes.
Human::Human(string nama, string nik, string gender)
{
    this->nama = nama;
    this->nik = nik;
    this->gender = gender;
}

// Getter and Setter
// set nama
void Human::setNama(string nama)
{
    this->nama = nama;
}

// get nama
string Human::getNama()
{
    return nama;
}

// set nik
void Human::setNik(string nik)
{
    this->nik = nik;
}

// get nik
string Human::getNik()
{
    return nik;
}

// set gender
void Human::setGender(string gender)
{
    this->gender = gender;
}

// get gender
string Human::getGender()
{
    return gender;
}

// Destructor
Human::~Human()
{
}
